package.json

```diff
@@ -313,6 +316,7 @@
     "express-winston": "^4.0.5",
     "file-type": "^16.5.4",
     "form-data": "^4.0.0",
+    "ftp": "^0.3.10",
```

Comment:
[optional improvement](https://www.npmjs.com/package/basic-ftp), "ftp" looks abandoned

Reply:
After further inspection, yes it does seem to be abandoned and has several variabilities. As such, I have switched to basic-ftp.